Cargo.toml

```
@@ -9,6 +9,13 @@ edition = "2018"
 repository = "https://github.com/ZcashFoundation/ed25519-zebra"
 description = "Zcash-flavored Ed25519 for use in Zebra."
 resolver = "2"
+include = [
+    "/src",
+    "/*.md",
+    "/LICENSE-*",
+    "/tests",
+    "/benches",
```

I'd avoid tests, benches and non-readme markdown files. They are useful in the repo, but not in the installed library since none of them are easily usable anyway when installed from crates.io.

Not having wildcards makes things a bit easier as well, avoiding accidental inclusion of unexpected files.

I would have expected this instead:
```suggestion
    "/README.md",
    "/LICENSE-APACHE",
    "/LICENSE-MIT",
```